nonce/nonce_test.go

```diff
@@ -192,11 +189,7 @@ func TestRemoteRedeem(t *testing.T) {
 
 	// Attempt to redeem a nonce with a prefix in the prefix map, remote returns invalid
 	// expect false, nil
```

Comment:
This comment is documenting the opposite of what the code below it does -- the comment says that the remote is returning "invalid", and the test is expecting "false", but in fact the remote is returning "valid" and the test is expecting "true".

The same inversion happens on lines 184/185.

We should either swap the comments, or swap the invalidRedeemer/validRedeemer in the prefixMap.